src/Range.tsx

```diff
@@ -32,10 +33,11 @@ class Range extends React.Component<IProps> {
   };
   trackRef = React.createRef<HTMLElement>();
   thumbRefs :React.RefObject<HTMLElement>[] = [];
+  resizeObserver: ResizeObserver;
   schdOnMouseMove: (e: MouseEvent) => void;
   schdOnTouchMove: (e: TouchEvent) => void;
   schdOnEnd: (e: Event) => void;
-  schdOnWindowResize: () => void;
```

Comment:
I like the `schdOnTrackResize` name but we should keep `schdOnWindowResize` as an alias for while so we don't have to introduce a breaking change. It seem pretty rare that you would resize the window without resizing the track at the same time so that should be fine.